integrationtest/osgi/karaf-it/src/test/java/org/hibernate/search/test/integration/osgi/HibernateSearchWithKarafIT.java

```diff
@@ -138,6 +148,24 @@ public void tearDown() {
 		bundleContext.ungetService( serviceReference );
 	}
 
+	/**
+	 * Starts all bundles to make sure there are no "uses constraint violations" caused by packages exported from the
+	 * bundles contributed by the HSEARCH Karaf feature.
+	 */
+	@Test
+	@TestForIssue(jiraKey="HSEARCH-1774")
+	public void testStartingAllBundles() throws Exception {
```

Comment:
Interesting. So the problem was that not all bundles are started by default. Nice test.